sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -385,8 +386,8 @@ class HealthcareEntity(DictMixin):
         :ivar int offset: The Healthcare entity text offset from the start of the document.
         :ivar float confidence_score: Confidence score between 0 and 1 of the extracted
             entity.
-        :ivar links: A collection of entity references in known data sources.
-        :vartype links: list[~azure.ai.textanalytics.HealthcareEntityLink]
+        :ivar data_sources: A collection of entity references in known data sources.
+        :vartype links: list[~azure.ai.textanalytics.HealthcareEntityDataSource]
```

Comment:
datasource?

Reply:
Updated!

Reply:
I think .net has it as `datasources`

Reply:
Oh oops, not yet.  I will update shortly.  :)